pkg/fiber-middleware/oapi_validate_test.go

```diff
@@ -0,0 +1,430 @@
+package middleware
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/getkin/kin-openapi/openapi3filter"
+	"github.com/gofiber/fiber/v2"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed test_spec.yaml
+var testSchema []byte
+
+func doGet(t *testing.T, app *fiber.App, rawURL string) *http.Response {
+	u, err := url.Parse(rawURL)
+	if err != nil {
+		t.Fatalf("Invalid url: %s", rawURL)
+	}
+
+	req := httptest.NewRequest("GET", u.RequestURI(), nil)
+	req.Header.Add("Accept", "application/json")
+	req.Host = u.Host
+
+	r, err := app.Test(req)
+	if err != nil {
+		t.Fatalf("Failed to test request, URL=%q: %v", rawURL, err)
+	}
+
+	return r
+}
+
+func doPost(t *testing.T, app *fiber.App, rawURL string, jsonBody interface{}) *http.Response {
+	u, err := url.Parse(rawURL)
```

Comment:
Add call to `t.Testing()`

Reply:
You mean `t.Helper()` right? That has been fixed